components/brave_rewards/browser/test/rewards_state_browsertest.cc

```diff
@@ -1034,10 +1040,19 @@ IN_PROC_BROWSER_TEST_P_(UpholdStateMachine, Migration) {
   const auto& to = std::get<1>(params);
 
   profile_->GetPrefs()->SetInteger("brave.rewards.version", 9);
-  rewards_service_->SetEncryptedStringState(kWalletUphold, from);
+  auto encrypted =
+      rewards_browsertest_util::EncryptPrefString(rewards_service_, from);
+  ASSERT_TRUE(encrypted);
+  profile_->GetPrefs()->SetString("brave.rewards.wallets.uphold", *encrypted);
 
   rewards_browsertest_util::StartProcess(rewards_service_);
-  EXPECT_EQ(rewards_service_->GetEncryptedStringState(kWalletUphold), to);
+
+  const auto uphold_pref =
+      profile_->GetPrefs()->GetString("brave.rewards.wallets.uphold");
+  auto decrypted = rewards_browsertest_util::DecryptPrefString(rewards_service_,
+                                                               uphold_pref);
+  ASSERT_TRUE(decrypted);
+  EXPECT_EQ(decrypted, to);
```

Comment:
I know that an `absl::optional<>` can be compared to a value, but still we might want to write `EXPECT_EQ(*decrypted, to);` instead.

Reply:
Yep, that's what I meant to write anyway - thanks!